tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -891,7 +919,7 @@ func (a *AwsProvider) getPublicSubnetId(infraID string, vpc *ec2.Vpc) (string, e
 	}
 
 	err = fmt.Errorf("could not find a public subnet in VPC: %v", *vpc.VpcId)
-	if !foundPublicSubnet {
+	if !foundSubnet {
 		err = fmt.Errorf("could not find a public subnet in a zone that supports %s instance type",
```

Comment:
error message can be changed to `could not find the required subnet in a zone that supports %s instance type`